### ISSUE TITLE
paddleOCR 文本识别微调预训练模型

### ISSUE THREAD

**Issue by dddxst** [PaddlePaddle/PaddleOCR#9579]
数据集 17w+的数据， 500 epoches

- 系统环境/System Environment：centos 7.9 + NVIDIA-SMI 515.43.04    Driver Version: 515.43.04    CUDA Version: 11.7 

- 版本号/Version：Paddle：2.4.2.post117   问题相关组件/Related components：ch_PP-OCRv3_rec

- 运行指令/Command Code：
 python3 -m paddle.distributed.launch --gpus '0,1,2,3'  tools/train.py -c configs/rec/PP-OCRv3/ch_PP-OCRv3_rec_distillation.yml -o Global.pretrained_model=/home/xap/project/PaddleOCR/tan/ch_PP-OCRv3_rec_train/best_accuracy

执行上述指令后训练完执行转为 推理模型：python3 tools/export_model.py -c configs/rec/PP-OCRv3/ch_PP-OCRv3_rec_distillation.yml -o Global.pretrained_model=/home/xap/project/PaddleOCR/output/rec_ppocr_v3_distillation/latest  Global.save_inference_dir=tmp

- 完整报错/Complete Error Message：
导出后的模型在tmp文件夹下，用Student或者Teacher模型给与推理，效果很差，还不如不训练的

**Comment by dddxst** [comment]
只是导出模型为推理模型，在验证集上是要比不训练好的，没有报错也没有警告，问题很多类似这样的：
**输入数据 比如**  ['/home/tan/dataset/ocr/2019年11月11日_17848.png']
**推理结果为**   [('年月日', 0.9260101318359375)]
在验证集上比预训练模型准，换数据就完全没法用了

**Comment by dddxst** [comment]
验证集的结果： output/rec_ppocr_v3_distillation/best_accuracy
打印如下
[2023/03/30 17:14:44] ppocr INFO: metric eval ***************
[2023/03/30 17:14:44] ppocr INFO: acc:0.20236626662488327
[2023/03/30 17:14:44] ppocr INFO: norm_edit_dis:0.7448604579479862
[2023/03/30 17:14:44] ppocr INFO: Teacher_acc:0.019457216547197145
[2023/03/30 17:14:44] ppocr INFO: Teacher_norm_edit_dis:0.45957994780894684
[2023/03/30 17:14:44] ppocr INFO: fps:1610.9402065232648

**Comment by dddxst** [comment]
[2023/03/30 17:20:34] ppocr INFO: metric eval ***************
[2023/03/30 17:20:34] ppocr INFO: acc:0.9822249632773844
[2023/03/30 17:20:34] ppocr INFO: norm_edit_dis:0.9972488908589155
[2023/03/30 17:20:34] ppocr INFO: Teacher_acc:0.9818044182511482
[2023/03/30 17:20:34] ppocr INFO: Teacher_norm_edit_dis:0.9973028273709597
[2023/03/30 17:20:34] ppocr INFO: fps:1587.8796136399399

**Comment by tink2123** [comment]
验证集上看起来提升很多，是不是验证集太简单了，并不符合真实使用场景。一般这个情况是因为训练数据较少或者偏简单造成过拟合，建议扩充一下训练数据，减小迭代轮次，减小学习率。另外验证集尽量使用真实场景的数据，方便挑选出精度最佳的模型。

**Comment by dddxst** [comment]
验证集和训练集都是这17w+的数据，随机抽取了20%作为验证数据，我对数据集字符做了统计，字符出现300次以下的1161个，300次及以上的148个，现在模型对某些特定字符识别不够准确，那我是不是只要拿这小部分识别不够准确的数据拿去做训练就可以，同时减小迭代轮次，减小学习率

**Comment by nissansz** [comment]
> 验证集上看起来提升很多，是不是验证集太简单了，并不符合真实使用场景。一般这个情况是因为训练数据较少或者偏简单造成过拟合，建议扩充一下训练数据，减小迭代轮次，减小学习率。另外验证集尽量使用真实场景的数据，方便挑选出精度最佳的模型。

> 你这个数据好像有点模糊，finetune的时候学习率调小了吗，也可以针对bad case合成一些数据加进去试试

rec resnet34模型微调hard case时，lr要设置为多少？
默认配置如下
Optimizer:
name: Adam
beta1: 0.9
beta2: 0.999
lr:
learning_rate: 0.0005
regularizer:
name: L2
